ocaml/xapi/xapi_globs.ml

```diff
@@ -974,6 +979,7 @@ module Resources = struct
 		"startup-script-hook", startup_script_hook, "Executed during startup";
 		"rolling-upgrade-script-hook", rolling_upgrade_script_hook, "Executed when a rolling upgrade is detected starting or stopping";
 		"xapi-message-script", xapi_message_script, "Executed when messages are generated if email feature is disabled";
+		"non-managed-pifs", non_managed_pifs, "Executed during PIF.scan to find out which NICs should not be managed by xapi";
```

Comment:
Would be good to have the default value as a commented-out like in xapi.conf, [for example](https://github.com/xapi-project/xen-api/blob/master/scripts/xapi.conf#L84)